### PR TITLE
Assignment 4 w140057

### PR DESCRIPTION
**Pull request by RicTer3637**

### CODE REVIEW

File: Assignment4/rterol-w140057/task07.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  results=[]
+  for s,p,o in g.triples((None,RDF.type,ns.Person)):
+    results.append((s,p,o))
+  for s,p,o in g.triples((None,RDF.type,ns.Researcher)):
+    results.append((s,p,o))
+  for s,p,o in g.triples((None,RDF.type,ns.Professor)):
+    results.append((s,p,o))
+  for s,p,o in g.triples((None,RDF.type,ns.PhDStudent)):
+    results.append((s,p,o))
+  return results
```
Comment (anaigmo) [inline Assignment4/rterol-w140057/task07.py:70]: that's very manual, how about when you don't know all the subclasses?

File: Assignment4/rterol-w140057/task07.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+ print(s)
+
+# TO DO
+# Visualize the results
+
+from rdflib.plugins.sparql import prepareQuery
+
+q1 = prepareQuery('''
+  SELECT ?subclass WHERE{
+    ?subclass rdfs:subClassOf ns:LivingThing.
```
Comment (anaigmo) [inline Assignment4/rterol-w140057/task07.py:42]: and the subclasses of `?subclass`?

**Comment by anaigmo** [comment]
You first need to clean the PR, leaving only tasks 6 and 7. Also, the name of the folder does not look that it follows the naming convention: `<username>-<studentID>/`. Apart from that and my comments above, the rest is looking good.

**Comment by dgarijo** [comment]
The last commit did not address the requests raised by @anaigmo

**Comment by RicTer3637** [comment]
I created a second pull request due to creating a new branch to focus on the changes, should I continue the pull here instead or is that okay?

**Comment by dgarijo** [comment]
I will close this one

**Pull request closed without merging**